src/appkit.cs

```diff
@@ -26941,15 +26943,15 @@ interface NSWorkspace : NSWorkspaceAccessibilityExtensions {
 
 		[Async]
 		[Export ("setDefaultApplicationAtURL:toOpenContentTypeOfFileAtURL:completionHandler:")]
-		void SetDefaultApplicationToOpenContentType (NSUrl applicationUrl, NSUrl url, [NullAllowed] Action<NSError> completionHandler);
+		void SetDefaultApplicationToOpenContentType (NSUrl applicationUrl, NSUrl url, [NullAllowed] Action<NSError?> completionHandler);
```

Comment:
While this is correct, it doesn't currently change anything in the generated code, because the generator doesn't understand nullability annotations in generic types.

Reply:
I know, but rgen does, and I need to update the delegates before we migrate.